globocom/directional-navigation
# Fix: Some minor bugs on navigation

## Summary 

We currently have an issue regarding of navigation when there's no selected element on page. This bug was resolved in this PR.

## How to test

- Click on `body` is selected on the demo page, the keys (up, down, left, right) should navigate remembering the last focused element.
